<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vcl::qt::MeshProcessingMainWindow</class>
 <widget class="QMainWindow" name="vcl::qt::MeshProcessingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="vcl::qt::MeshViewer" name="meshViewer"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenMesh"/>
    <addaction name="actionSaveMeshAs"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenMesh">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open Mesh</string>
   </property>
  </action>
  <action name="actionSaveMeshAs">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save Mesh As...</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vcl::qt::MeshViewer</class>
   <extends>QWidget</extends>
   <header>vclib/ext/qt/mesh_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
